### PR TITLE
Adds DemoCommunityScreen. Adds ListItem component.

### PR DESCRIPTION
**Pull request by kateinkim**
## Please verify the following:

- [x] `yarn ci:test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

### Community Screen
- adds and styles new screen
- adds links to list items

### ListItem Component
- adds new component for displaying list-items
- adds demo for ListItem

## Photos

| ListItem | Updated DemoCommunityScreen |
|---|---|
| yulolimum-capture-2022-08-18--16-00-06@2x | CleanShot 2022-08-19 at 14 18 46@2x|

| Linking | KR translation for links |
|---|---|
CleanShot 2022-08-19 at 14 15 39|CleanShot 2022-08-19 at 14 29 25@2x|
|  |  |

### CODE REVIEW

File: boilerplate/app/navigators/DemoNavigator.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export function DemoNavigator() {
 
       <Tab.Screen
         name="DemoContributing"
```
Comment (yulolimum) [inline boilerplate/app/navigators/DemoNavigator.tsx:58]: We should probably this accordingly. TabBar too?

**Comment by leonskim** [comment]
Added `ko` translation for Community screen.

**Comment by mazenchami** [comment]
Updated ar.json
simulator_screenshot_2DB6E239-2397-41C4-9AFF-FCD8F6DE71E8

File: boilerplate/app/screens/DemoComponentsScreen/DemoComponentsScreen.tsx [hunk 2]
```diff
@@ -92,14 +83,12 @@ export function DemoComponentsScreen(props: DemoTabScreenProps<"DemoComponents">
                   {item.name}
                 </Text>
                 {item.useCases.map((u, index) => (
-                  <Pressable
-                    onPress={() => handleScroll(sectionIndex, index + 1)}
+                  <ListItem
```
Comment (mazenchami) [inline boilerplate/app/screens/DemoComponentsScreen/DemoComponentsScreen.tsx:86]: this is so much cleaner! 🎉

**Review by mazenchami — APPROVED** [review]

File: boilerplate/app/navigators/DemoNavigator.tsx [hunk 3]
```diff
@@ -5,16 +5,14 @@ import { TextStyle, ViewStyle } from "react-native"
...
-import { DemoPodcastListScreen } from "../screens/DemoPodcastListScreen"
+import { DemoCommunityScreen, DemoComponentsScreen, DemoDebugScreen } from "../screens"
 import { colors, typography } from "../theme"
 import { AppStackParamList, AppStackScreenProps } from "./AppNavigator"
 
 export type DemoTabParamList = {
   DemoComponents: undefined
   DemoDebug: undefined
-  DemoPodcastList: undefined
-  DemoContributing: undefined
```
Comment (yulolimum) [inline boilerplate/app/navigators/DemoNavigator.tsx:17]: @kateinkim what happened here? We want the podcast screen

File: boilerplate/app/navigators/DemoNavigator.tsx [hunk 4]
```diff
@@ -66,19 +64,8 @@ export function DemoNavigator() {
...
       <Tab.Screen
-        name="DemoPodcastList"
-        component={DemoPodcastListScreen}
-        options={{
-          tabBarLabel: translate("demoNavigator.podcastListTab"),
-          tabBarIcon: ({ focused }) => (
-            <Icon icon="menu" style={{ tintColor: focused && colors.tint }} />
-          ),
-        }}
-      />
```
Comment (yulolimum) [inline boilerplate/app/navigators/DemoNavigator.tsx:77]: same here. we're keeping this

File: boilerplate/app/navigators/DemoNavigator.tsx (hunk 3 above)
Comment (kateinkim) [reply boilerplate/app/navigators/DemoNavigator.tsx:17]: Did I delete this? I didn't have any merge conflict or anything. I don't know what's going on. My recent pull doesn't have these either.

**Review by yulolimum — COMMENTED** [review]
@kateinkim I think you might have deleted some stuff from Jamon's branch. Can you take a look? Also, the tests are failing.

File: boilerplate/app/i18n/ar.ts [hunk 5]
```diff
@@ -32,6 +32,7 @@ const ar: TxType = {
     componentsTab: "عناصر",
     debugTab: "تصحيح",
     communityTab: "واصل اجتماعي",
+    podcastListTab: "البودكاست",
```
Comment (kateinkim) [inline boilerplate/app/i18n/ar.ts:35]: Added Arabic translation. I don't know if it's correct. Just didn't want to bother Mazen on Friday evening.

Comment (jamonholmgren) [reply boilerplate/app/i18n/ar.ts:35]: @mazenchami

Comment (mazenchami) [reply boilerplate/app/i18n/ar.ts:35]: @kateinkim love it! some minor adjustments pushed

**Pull request merged**